.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -32,12 +34,6 @@ if [[ -n "$BUILDKITE_PULL_REQUEST" && -n "$VERSION_QUALIFIER" ]] ; then
     exit 2
 fi
 
-# Tests must be run in PR builds
-if [[ -n "$BUILDKITE_PULL_REQUEST" && "$RUN_TESTS" = false ]] ; then
-    echo "RUN_TESTS should not be false PR builds"
-    exit 3
-fi
-
 echo "environment variables:"
 env
```

Comment:
This should only be temporary. Printing every environment variable could give away sensitive information. Please remove this once debugging is complete.